src/app/shared/ui/header/header.component.html

```diff
@@ -0,0 +1,23 @@
+<header>
+  <div class="header__close-modal-button">
+    <mat-toolbar color="primary">
+      <button mat-icon-button class="header__menu-btn" aria-label="Example icon-button with menu icon example-icon">
```

Comment:
`aria-label` do wyrzucenia - to chyba jakieś pozostałości po przekopiowaniu z Materiala